__tests__/app.test.js

```diff
@@ -148,6 +148,26 @@ describe("/api/articles/:article_id", () => {
         expect(body.msg).toBe('Bad Request')
     })
     });
+    test('GET: 200, should provide client with a comment_count property that represents the total count of all the comments with the given article_id', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+            const { article } = body;
+            expect(article).toEqual({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 100,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                  comment_count: 11
```

Comment:
I don't think you need to check all the other properties, this is just for comment_count right? So just checking that the response has the comment_count property is enough :)